Anaconda-Platform/anaconda-client
# Fix `anaconda config --set` when BINSTAR_CONFIG_DIR is set

Caused by #369 

To test:

```
BINSTAR_CONFIG_DIR=. anaconda config --set url http://localhost:5000
```

should create a file `data/config.yaml`.
